main.py

```diff
@@ -0,0 +1,82 @@
+"""
+changelog:
+    3/3/20: Finished implementation of ctrladd, fixed various bugs in actual code (but not pseudocode), changed
+        qubits generation, rewrote main, created changelog and to do.
+    2/26/20: Wrote out ctrladd function in pseudocode, implemented through step 4
+    2/25/20: Wrote basic code to create qubits
+todo:
+    write ctrladd function (DONE)
+    move qubit creation to separate function
+    fix ancillary qubit location (and its use in ctrladd)
+    write pseudocode for the multiplier portion
+    implement pseudocode
+    write testing function (displays two summands and sum, two factors and product)
+
+"""
+
+import cirq
+
+# testing qubit setup
+# circuit.append(cirq.H(q) for q in qubitsA)
+# circuit.append(cirq.Z(q) for q in qubitsB)
+# circuit.append(cirq.X(q) for q in qubitsOut)
+
+# def the following as ctrl_add with ctrl as the ctrl, qubitsSumnA as the bits of one summand, and qubitsSumnB as the
+# bits of the other, with the result stored in qubitsSumnB. qubitsSumnA[n] is ancillary, qubitsSumnB[n] = s[4] dot ctrl
+def ctrl_add (ctrl, qubitsSumnA, qubitsSumnB, n, circuit):      # circuit is possibly unnecessary, n is definitely unnecessary
```

Comment:
make `circuit` the first parameter

Reply:
I think it would be best to infer `n` from `qubitsSumB` and not pass it as parameter